e2e/helpers/browser_helper.js

```diff
@@ -100,6 +104,28 @@ module.exports = class BrowserHelpers extends Helper {
     }
   }
 
+  async grabAttribute(locator, attr) {
+    const elements = await this.locateSelector(locator);
+
+    let getAttribute;
+
+    if(this.isPuppeteer()){
+      getAttribute = async (element, attr) =>  (await element.getProperty(attr)).jsonValue();
+    } else {
+      getAttribute = async (element, attr) => await element.getAttribute(attr);
+    }
+
+    const texts = elements.map(async element => getAttribute(element, attr));
+
+    if (texts.length > 1) {
+      throw new Error(`More than one element found for locator ${locator}`);
```

Comment:
It seems that both the [puppeteer](https://github.com/codeceptjs/CodeceptJS/blob/feadbd830c2d85c2f62f65700ebc60638900c58e/lib/helper/Puppeteer.js#L1765) and [webdriver](https://github.com/codeceptjs/CodeceptJS/blob/feadbd830c2d85c2f62f65700ebc60638900c58e/lib/helper/WebDriver.js#L1271) implementation just warn that the first value is being used in the debug log. Do we want to maintain that functionality instead of throwing an error?

Reply:
i think that for us it is an error situation, in case multiple elements matches we should prepare specific selector that identifies exact element